src/Common/Types.ts

```diff
@@ -51,11 +51,12 @@ export interface Teams {
 export enum CHECKBOX_VALUE {
     CHECKED = 'CHECKED',
     INTERMEDIATE = 'INTERMEDIATE',
+    BULK_CHECKED = 'BULK_CHECKED',
 }
 export interface CheckboxProps {
     onChange: (event) => void
     isChecked: boolean
-    value: 'CHECKED' | 'INTERMEDIATE'
+    value: 'CHECKED' | 'INTERMEDIATE' | 'BULK_CHECKED'
```

Comment:
We should use `CHECKBOX_VALUE` here

Reply:
Yes, will add a comment here, initially had used enum and removed all hardcoding from dashboard but there are still few places in dashboard that passively derives this value to be a string through prop drilling so hadn't changes those since that will have larger impact to test.